Manifests/ManagedPreferencesApplications/de.fau.rrze.NetworkShareMounter.plist

```diff
@@ -247,9 +365,11 @@ A profile can consist of payloads with different version numbers. For example, c
 		</dict>
 		<dict>
 			<key>pfm_description</key>
-			<string>Array with all network shares. Example: smb://filer.your.domain/share. Note: %USERNAME% will be replaced with the login name of the current user.</string>
+			<string>⚠️ Deprecated with version 3. Value will be removed in a future version of the Network Share Mounter.</string>
```

Comment:
For reference and for users on older versions, we usually keep the description of deprecated keys. Could you please revert the changes to this line?

You can then mark the key as deprecated by using `pfm_app_deprecated` set to `3.0.0`.

Reply:
Please also make sure to add a `pfm_app_deprecated` tag set to `3.0.0`, thanks!

Reply:
My bad, I see now that it was already added previously.